python/lsst/jointcal/utils.py

```diff
@@ -97,26 +98,40 @@ def compute(catalogs, calibs):
             # will change if the data_refs are ordered differently.
             # All the more reason to use a proper n-way matcher here. See: DM-8664
             refcat = catalogs[0]
-            refcalib = calibs[0]
+            refcalib = photoCalibs[0] if photoCalibs != [] else None
             dist_rel, flux_rel, ref_flux_rel, source_rel = self._make_match_dict(refcat,
                                                                                  catalogs[1:],
-                                                                                 calibs[1:],
+                                                                                 photoCalibs[1:],
```

Comment:
This was a good question, and I had to think about it a bit. photoCalibs is never `None`, but it can be `[]`. In that case, `photoCalibs[1:]==[]`, so everything works fine.